NYCJobsWeb/Controllers/HomeController.cs

```diff
@@ -72,10 +72,13 @@ public ActionResult Suggest(string term, bool fuzzy = true)
                 suggestions.Add(result.Text);
             }
 
+            // Get unique items
+            List<string> uniqueItems = suggestions.Distinct().ToList();
```

Comment:
@liamca Was this just a bug fix, or did you need to do this because of the SDK upgrade?